python/graphscope/config.py

```diff
@@ -114,3 +114,5 @@ class GSConfig(object):
 
     # download_retries
     dataset_download_retries = 3
+
+    hosts = ["localhost"]
```

Comment:
The hosts argument is not suitable as a configuration entry.

Reply:
Removed